x-pack/plugins/security_solution/cypress/integration/fields_browser.spec.ts

```diff
@@ -94,9 +95,10 @@ describe('Fields Browser', () => {
           cy.get(FIELDS_BROWSER_SYSTEM_CATEGORIES_COUNT)
             .invoke('text')
             .then((systemCategoriesCount) => {
-              cy.get(FIELDS_BROWSER_FIELDS_COUNT)
-                .invoke('text')
-                .should('eq', `${+hostCategoriesCount + +systemCategoriesCount} fields`);
+              cy.get(FIELDS_BROWSER_FIELDS_COUNT).should(
+                'have.text',
+                `${+hostCategoriesCount + +systemCategoriesCount} fields`
```

Comment:
Nit: I don't think you need the unary on the second argument as it should be coerced to the type of the first: 
```suggestion
                `${+hostCategoriesCount + systemCategoriesCount} fields`
```

Reply:
If I do your suggested change the test fails :(

Reply:
Yep, I was wrong. Sorry about that! 🤦